README.md

```diff
@@ -78,27 +176,29 @@ So here are three default placeholders for your file system:
 
 - `~` Your **home** directory
 - `.` The **current** directory
-- `..` The directory in which your current directory is contained—referred to as the "**parent**"
-directory.
+- `..` The directory in which your current directory is contained—referred to
+as the "**parent**" directory.
 
 You can supply any path to the `cd` command to nkellyegreenegate to that location.
 
 ## Demonstrate Manipulating Files in the Command Line
 
 ### Paths in Shell
 
-The path supplied to the `cd` command, for example `/Users/kellyegreene`, is known as an absolute
-path.
+The path supplied to the `cd` command, for example `/Users/kellyegreene`, is
```

Comment:
swap the order of these:

The path supplied to the `cd` command can be either *absolute* or *relative* paths. An absolute path is a path that always gets you to the same folder. You can recognize them because they start with `/`. For example `/Users/kellyegreene`, is an absolute path.